vignettes/release-checklists.qmd

```diff
@@ -51,10 +52,19 @@ Parts of this document is adapted for The Hubverse from [The Carpentries Develop
 
 ### Release to CRAN
 
-The release process for CRAN releases requires extra time and attention.
-
- - [ ] run `devtools::release()` and follow the prompts
- - [ ] after CRAN submission, add and commit the `CRAN_SUBMISSION` file
+The release process for CRAN releases requires extra time and attention. Make sure to familiarise yourself with [the steps to release to CRAN](#release-to-cran). A good place to start would be to 
+
+ - [ ] open a template release issue using `usethis::use_release_issue()` and follow the instructions.
+ - [ ] If this the first release on CRAN, make sure to:
+    - [ ] Add the CRAN installation command to the installation instructions in the `README.Rmd`.
+    - [ ] Add a CRAN badge to the `README.Rmd` using `usethis::use_cran_badge()`.
+    = [ ] Rebuild `README.md`.
+ - [ ] Use `devtools::check(remote = TRUE, manual = TRUE)` to check the package locally.
```

Comment:
OMG I didn't know about `remote = TRUE`! (and it's been around since 2016?!?!!? https://github.com/r-lib/devtools/issues/1271) 🤯